.github/workflows/gh-runner-release.yml

```diff
@@ -38,6 +39,6 @@ jobs:
           context: ./infrastructure/images/gh-runner
           push: true
           platforms: linux/amd64,linux/arm64
-          tags: ghcr.io/${{ github.repository }}/gh-runner:v${{ steps.version.outputs.version }}
+          tags: ghcr.io/${{ stepd.vars.outputs.reponame }}/gh-runner:v${{ steps.vars.outputs.version }}
         env:
           SOURCE_DATE_EPOCH: ${{ env.TIMESTAMP }}
```

Comment:
```suggestion
          SOURCE_DATE_EPOCH: ${{ env.TIMESTAMP }}
```

Reply:
"Typo" while doing review on phone?

Reply:
New line at the end of the file :) It's just a nit.

Reply:
I like nits!